.github/workflows/test.yml

```diff
@@ -361,31 +361,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - uses: ./.github/actions/setup-turborepo-environment
-        with:
-          target: ${{ matrix.os.name }}
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-      - name: Setup Pnpm
-        uses: pnpm/action-setup@v2.2.4
+      - name: "Setup Node"
+        uses: ./.github/actions/setup-node
         with:
-          version: 7.2.1
+          enable-corepack: "false"
+          package-install: "false"
 
-      - name: Make sure pnpm always has a cache
+      - name: Install Turborepo globally
         shell: bash
-        run: |
-          mkdir -p `pnpm store path`
```

Comment:
We lose this, but it looks like conditionally making this happen causes issues with the setup-node Post cleanup code: https://github.com/vercel/turbo/actions/runs/5006261750/jobs/8971325789?pr=4929

> Error: Cache folder path is retrieved for pnpm but doesn't exist on disk: /home/runner/setup-pnpm/node_modules/.bin/store/v3

I _think_ we can setup pnpm cache correctly in a separate pass, it's mostly incidental that the Examples job has this line and the others jobs don't.

Reply:
reverting this piece, something isn't working in CI and I don't want to spend more time on this right now!